docs/page/css/syntax.css

```diff
@@ -77,3 +77,6 @@
 .highlight .vg { color: #008080 } /* Name.Variable.Global */
 .highlight .vi { color: #008080 } /* Name.Variable.Instance */
 .highlight .il { color: #009999 } /* Literal.Number.Integer.Long */
+
+/* Missing YAML styles */
+.highlight .l-Scalar-Plain { color: #008080 }
```

Comment:
What is this line doing? Why does `YAML` equate to `l-Scalar-Plain`? Is `YAML` unknown to the highlighter?